### PR TITLE
fix(ui): change to fit new ui

### PR DESCRIPTION
**Pull request by ororsatti**

### CODE REVIEW

File: packages/app-connector/src/adapters/iframe.ts [hunk 1]
```diff
@@ -58,7 +58,8 @@ export function connect(
   dialog.style.borderWidth = "0px";
   dialog.style.borderRadius = "16px";
   dialog.style.padding = "0px";
-  dialog.style.backgroundColor = "#19473f";
+  dialog.style.backgroundColor = "white";
```
Comment (ororsatti) [inline packages/app-connector/src/adapters/iframe.ts:61]: relevant change

File: packages/app-connector/src/adapters/iframe.ts [hunk 2]
```diff
@@ -106,6 +107,9 @@ export function connect(
   // Create the iframe that will host the client
   const iframe = document.createElement("iframe");
   const sandboxAttr = document.createAttribute("sandbox");
+  const scrollingAttr = document.createAttribute("scrolling");
```
Comment (ororsatti) [inline packages/app-connector/src/adapters/iframe.ts:110]: relevant change

**Comment by rrrliu** [comment]
@ororsatti I think you need to run `pnpm i` https://github.com/proofcarryingdata/parcnet-client/actions/runs/11486162462/job/31968839684?pr=22

**Review by robknight — APPROVED** [review]

**Comment by robknight** [comment]
lgtm!

**Pull request merged**